client/signup/steps/site-type/form.jsx

```diff
@@ -40,52 +36,34 @@ class SiteTypeForm extends Component {
 		};
 	}
 
-	handleRadioChange = event => this.setState( { siteType: event.currentTarget.value } );
-
-	handleSubmit = event => {
-		const { siteType } = this.state;
-
-		event.preventDefault();
-
+	handleSubmit = type => {
 		this.props.recordTracksEvent( 'calypso_signup_actions_submit_site_type', {
-			value: siteType,
+			value: type,
 		} );
 
-		this.props.submitForm( siteType );
+		this.setState( { siteType: type } );
```

Comment:
This `siteType` state was used by the radio buttons to track selection. Now that they are gone, it can be removed entirely, from all the `setState()` call and the constructor.

Reply:
We still need it to track selection for Jetpack sites.